packages/node-cli/src/commands/ManageAccountCommandGenerator.js

```diff
@@ -0,0 +1,271 @@
+/*
+ ** Copyright (c) 2020 Oracle and/or its affiliates.  All rights reserved.
+ ** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+ */
+"use strict";
+
+const BaseCommandGenerator = require("./BaseCommandGenerator");
+const SdkExecutionContext = require("../SdkExecutionContext");
+const { executeWithSpinner } = require("../ui/CliSpinner");
+const SdkOperationResultUtils = require("../utils/SdkOperationResultUtils");
+const CommandUtils = require("../utils/CommandUtils");
+const NodeTranslationService = require("../services/NodeTranslationService");
+const ManageAccountOutputFormatter = require("./outputFormatters/ManageAccountOutputFormatter");
+const AuthenticationService = require("../core/authentication/AuthenticationService");
+const { ManageAccountActionResult } = require("./actionresult/ManageAccountActionResult");
+const assert = require("assert");
+
+const inquirer = require("inquirer");
+
+const {
+   COMMAND_MANAGE_ACCOUNT: { ERRORS, QUESTIONS, QUESTIONS_CHOICES, MESSAGES },
+   YES,
+   NO,
+} = require("../services/TranslationKeys");
+
+const ACTION = {
+   EXIT: "exit",
+   RENAME: "rename",
+   REMOVE: "remove",
+   REVOKE: "revoke",
+};
+
+const COMMAND = {
+   OPTIONS: {
+      INFO: "info",
+      LIST: "list",
+      REMOVE: "remove",
+      RENAME: "rename",
+      RENAMETO: "renameto",
+   },
+   FLAGS: {
+      NO_PREVIEW: "no_preview",
+      SKIP_WARNING: "skip_warning",
+      VALIDATE: "validate",
+   },
+};
+
+const ANSWERS_NAMES = {
+   SELECTED_AUTH_ID: "selected_auth_id",
+   ACTION: "action",
+   AUTHID: "authId",
+   RENAMETO: "renameto",
+   REMOVE: "remove",
+};
+
+const {
+   showValidationResults,
+   validateAuthIDNotInList,
+   validateAlphanumericHyphenUnderscore,
+   validateFieldHasNoSpaces,
+   validateFieldIsNotEmpty,
+   validateMaximumLength,
+   validateSameAuthID,
+} = require("../validation/InteractiveAnswersValidator");
+
+const PROPERTIES = {
+   INFO: "info",
+   ACCOUNT_INFO: "accountInfo",
+   URLS: "urls",
+};
+
+const DOMAIN = "domain";
+module.exports = class ManageAccountCommandGenerator extends BaseCommandGenerator {
+   constructor(options) {
+      super(options);
+      this._authenticationService = new AuthenticationService(options.executionPath);
+      this._outputFormatter = new ManageAccountOutputFormatter(options.consoleLogger);
+   }
+
+   async _getCommandQuestions(prompt) {
+      const authIDList = await this._authenticationService.getAuthIds(this._sdkExecutor);
+      let answers = await this._selectAuthID(authIDList, prompt);
+      this._outputFormatter.logAccountInfo(answers[ANSWERS_NAMES.SELECTED_AUTH_ID]);
```

Comment:
the OutputFormatter should take care of the output of the command, no? In this case, we're asking the questions (input of the command). I don't think you should use outputFormatter here. Maybe it could be an option to extract `logAccountInfo` to an utility class, and call it both here and in OutputFormatter (if it's needed)

Reply:
We decided to mantain for the moment since it's an output for non-interactive -info option, and also it's a message shown in the console in the middle of interactive (not final output though, but don't be willing to overcomplicate by creating a new service)